Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -352,6 +346,19 @@ def start(self):
 
         return shell.Compile.start(self)
 
+    def buildCommandKwargs(self, warnings):
+        kwargs = super(CompileWebKit, self).buildCommandKwargs(warnings)
+        # https://bugs.webkit.org/show_bug.cgi?id=239455: The timeout needs to be >20 min to
+        # work around log output delays on slower machines.
+        # https://bugs.webkit.org/show_bug.cgi?id=247506: Only applies to Xcode 12.x.
+        if 'timeout' in kwargs:
```

Comment:
probably don't need this check. Even if buildbot set the timeout (e.g: default timeout), we probably want to override that.